### ISSUE TITLE
Multiple Headers insertion

### ISSUE THREAD

**Issue by blackode** [samuelmanzanera/ex_license#2]
# Description
When we run the mix task multiple times it keeps on adding header lines.

## How to recreate
- Add two different licenses then repeat the same

```sh
mix insert_license --license mit
mix insert_license --license boost
mix insert_license --license mit
```

![image](https://user-images.githubusercontent.com/9107477/158044345-44bafe22-3338-4b28-ad93-8f90f99ade7c.png)

**Comment by samuelmanzanera** [comment]
Thanks for the issue 👍 
I will look to it .

**Comment by ghost** [comment]
Closed by d78bb798003f760e9761d238d6f41c341bd2cb4b
